vendor/github.com/vulcand/oxy/forward/fwd.go

```diff
@@ -335,6 +303,7 @@ func (f *websocketForwarder) copyRequest(req *http.Request, u *url.URL) (outReq
 
 	outReq.URL = utils.CopyURL(req.URL)
 	outReq.URL.Scheme = u.Scheme
+	outReq.URL.Path = outReq.RequestURI
```

Comment:
Thx for your report, the problem is already fix, just need to be merge and release
https://github.com/containous/oxy/pull/22